specs/flashtestations.md

```diff
@@ -444,12 +428,34 @@ This operation returns the raw attestation quote that was used to register the a
 
 ### Key Requirements
 
-1. **Single Current Endorsement**: An address has exactly one current endorsement bundle for each workloadId it's registered under.
-2. **Complete Revocation**: When an endorsement bundle becomes invalid, all addresses registered with that specific (workloadId, tcbHash) combination are removed completely - there is no partial revocation.
-3. **Quote Storage**: The system maintains a copy of the most recent attestation quote used to register each address, supporting external verification and auditability.
+1. **Simple Storage Model**: The registry maintains a simple mapping between addresses, workloadIds, and their attestation quotes without tracking complex endorsement relationships.
+
+2. **Individual Verification**: Instead of batch removal based on endorsement bundles, attestations are verified individually when needed, allowing for more granular management.
+
+3. **Quote Storage**: The system maintains a copy of the attestation quote used to register each address, supporting external verification and auditability.
 
 4. **Gas Efficiency**: The lookup operation must be extremely efficient (O(1)) regardless of the number of addresses stored.
 
+### Attestation Verification Endpoint
```

Comment:
Is this an offchain webserver? That's what I think of when I think endpoint. Further down the spec says "If verification fails, it marks the address as outdated in the registry", which makes me think this is an onchain function. Can you add a Solidity function signature for this, if that last sentence is correct?